### PR TITLE
SQ-51/Prepare for proper EventDetails viewmodel

### PR DESCRIPTION
**Pull request by rock3r**
* Remove unused stuff, move Speaker module to the right place
 * Use ExperienceLevel in Event model and stuff
 * Create EventDetails viewmodel (not used yet, really, as we miss most of the data so far, but NEXT PR)

### CODE REVIEW

File: app/src/main/java/net/squanchy/eventdetails/widget/EventDetailsLayout.java [hunk 1]
```diff
@@ -51,7 +51,7 @@ public void updateWith(Event event) {
         placeView.setVisibility(event.placeVisibility());
         placeView.setText(event.place());
         speakersView.setVisibility(event.speakersVisibility());
-        speakersView.setText(event.speakers());
+        speakersView.setText(event.speakersNames());
```
Comment (tiwiz) [inline app/src/main/java/net/squanchy/eventdetails/widget/EventDetailsLayout.java:54]: Just as a reminder, we have to add them in the database, from the JSON, we have the indexes of the speakers

File: app/src/main/java/net/squanchy/schedule/domain/view/Event.java [hunk 2]
```diff
@@ -18,23 +17,23 @@ public static Event create(
...
+                .speakersNames(speakersNamesStringFrom(speakerNames))
+                .speakersVisibility(speakerNames.isEmpty() ? View.GONE : View.VISIBLE)
                 .trackVisibility(View.GONE) // todo add track
-                .experienceLevelIcon(experienceLevelIconFor(experienceLevel))
+                .experienceLevel(experienceLevel)
                 .build();
     }
 
-    private static String speakerNames(List<String> speakers) {
+    private static String speakersNamesStringFrom(List<String> speakers) {
```
Comment (tiwiz) [inline app/src/main/java/net/squanchy/schedule/domain/view/Event.java:36]: Once more, we have to edit the model on Firebase, I guess

**Review by tiwiz — APPROVED** [review]
I think we can merge, just take a look at the comments :-)

File: app/src/main/java/net/squanchy/eventdetails/widget/EventDetailsLayout.java (hunk 1 above)
Comment (rock3r) [reply app/src/main/java/net/squanchy/eventdetails/widget/EventDetailsLayout.java:54]: This is just a name change. Model changes are pending review from you and @fourlastor

File: app/src/main/java/net/squanchy/schedule/domain/view/Event.java (hunk 2 above)
Comment (rock3r) [reply app/src/main/java/net/squanchy/schedule/domain/view/Event.java:36]: Ditto

**Pull request merged**